### ISSUE TITLE
视频笔记: 台大李宏毅2021春季深度学习课程 (60h)

### ISSUE THREAD

**Issue by nonocast** [nonocast/me#206]
Prerequisite
- Math
  - Calculus (微积分), Linear algebra (线性代数) and Probability (概率)
- Programming
  - Python
- Hardware
  - Google Colab

Machine Learning ≈ Looking for Function
- Speech Recognition: f(声音片段) = "How are you"
- Image Recognition: f(猫的图片) = "Cat"

Different types of Functions
- Regression: The function outputs a scalar. 比如: f(PM2.5 today, temperature, Concentration of O3) = PM2.5 of tomorrow
- Classification: Given options (classes), the function outputs the current one.比如: 判断垃圾邮件 (Y/N), Alpha Go就是输入当前棋盘棋子的位置，输出下一步位置，从19x19的选择中选择
- Generation (Structured Learning) : create  something with structure (image ,document)

**Model** y = b+wx1
- Model就是带有位置参数的function
- w: weight
- b: bias
- w和b从训练获取

Define Loss
- 预测值和实际值差距的sigma/N, 数值越大偏差越大
- L is mean absolute error (MAE), square error (MSE) 

hyperparameters: 超参
- 为了找到w*, b*过程中所需要的参数，比如learning rate

Machine Learning 三步:
1. function with unknow
2. define loss from training data
3. optimization
